### PR TITLE
Update to 1.17-pre1, Java 16

### PR DESCRIPTION
**Pull request by spiralhalo**
Tested in production

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     ]
   },
   "depends": {
-    "fabricloader": ">=0.4.0",
-    "fabric": "*"
+    "fabricloader": ">=0.11.3",
+    "fabric": "*",
+    "java": ">=16"
```
Comment (UpcraftLP) [inline src/main/resources/fabric.mod.json:28]: This should probably also require MC 1.17 snapshots and up

Comment (spiralhalo) [reply src/main/resources/fabric.mod.json:28]: You're right. Correcting it now.

**Review by Pyrofab — APPROVED** [review]

**Comment by doctor4t** [comment]
Yo that's pog, will merge when I can

**Comment by spiralhalo** [comment]
Nice. Works on 1.17-pre3 without changes.

**Review by UpcraftLP — APPROVED** [review]

**Pull request merged**